src/event.cc

```diff
@@ -258,21 +261,28 @@ EventScheduler::clean()
 }
 
 void
-EventScheduler::dump(Packable *out)
+EventScheduler::dump(std::ostream &yaml)
 {
+    static const SBuf indent("  ",2);
+
     if (last_event_ran)
-        out->appendf("Last event to run: %s\n\n", last_event_ran);
+        yaml << "last event to run: " << last_event_ran << "\n";
```

Comment:
The current wording is not wrong but is ambiguous. Since we are changing this key, I suggest:

```suggestion
        yaml << "last event that ran: " << last_event_ran << "\n";
```

```suggestion
        yaml << "last event: " << last_event_ran << "\n";
```

... or something like that.